engine/src/main/scala/cromwell/engine/backend/package.scala

```diff
@@ -6,8 +6,6 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.collection.immutable.Seq
 
 package object backend {
-  class WorkflowContext(val root: String)
```

Comment:
eh, I see - all the stuff I commented on above was just moved from a separate package object. blech. disregard above unless you want to be amazing :)